### PR TITLE
update overview for crud-http tutorial extension

### PR DESCRIPTION
**Pull request by alisaduncan**
Changed how CRUD, HTTP, REST are introduced to hopefully make it easier to absorb. Added an example to illustrate the relationship between HTTP and REST. 

@pelcia would you mind taking a look to ensure this still fits your vision?

### CODE REVIEW

**Comment by alisaduncan** [comment]
Oops! I meant @pelagia123!

**Review by pelagia123 — APPROVED** [review]

**Comment by alisaduncan** [comment]
@willi84 @pelagia123 PR updated 😀

File: crud-and-http/CRUD_HTTP.md [hunk 1]
```diff
@@ -36,16 +36,16 @@ Responses are made up of
...
+Let's consider an example of how **REST** and **HTTP** relate to each other. Imagine **HTTP** is an oven used for baking cakes - in other words it's a foundation for baking yummy treats. 😋 And let's imagine a cake represents data. 🍰 **REST** is like a baking pan used to hold the cake in a shape - if a baker doesn't use a cake pan, the cake won't have shape. Imagine **REST** is a round cake pan, a shape everyone is familiar with and bakers know how to decorate. **REST** is a common "language" that round cakes have so that all bakers have standard tools to work with them and have experience working with.
```
Comment (pelagia123) [inline crud-and-http/CRUD_HTTP.md]: ❤️

**Review by pelagia123 — APPROVED** [review]

**Comment by willi84** [comment]
I love this PR. we need more from IT. Less using buzzwords.

**Pull request merged**